src/olympia/devhub/templates/devhub/verify_email.html

```diff
@@ -98,8 +98,8 @@ <h3>{{ _('Success!') }}</h3>
 {% with submit_text=button_text %}
   {% include 'devhub/verify_email_form.html' %}
 {% endwith %}
-{% trans %}
-  Having trouble? Please <a href="mailto:{{ support_email }}?subject={{ support_subject }}&body={{ support_body }}">contact AMO Admins</a> if you need help.
+{% trans a_attrs='target="_blank" rel="noopener noreferrer" href="%s/documentation/publish/developer-accounts/#email-issues?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=devhub"'|format(settings.EXTENSION_WORKSHOP_URL)|safe%}
```

Comment:
Nit: If you defined the link href at least in the view, we could test it against an absolute link. This would also make verifying easier. Could also use urllib.join for a more "python knows the way" construction of the url itself.

```python
    path = 'documentation/publish/developer-accounts/#email-issues'
    query_params = {
        'utm_source': 'addons.mozilla.org',
        'utm_medium': 'referral',
        'utm_content': 'devhub'
    }
    
    query_string = urlencode(query_params)
    full_path = f"{path}?{query_string}"
    help_link = urljoin(settings.EXTENSION_WORKSHOP_URL, full_path)
```

Reply:
I copy pasted this from other places in AMO. If we want to update that, we should update it everywhere (which would be out of scope for this PR), rather than having yet another different solution for adding a link to EW. If you grep for `_blank`, you'll see that we already have a bunch of different implementations, including some where we even hard-code the hostname.

Reply:
FWIW, I don't think many examples of bad code is a reason for more bad code. That is a pretty weak argument. Also, we don't have to update 100% of a bad pattern at once or not at all.

This is not a must have, just makes the code more reliant on standards based tools for building the URL and IMO more readable.